manager/metrics/collector.go

```diff
@@ -0,0 +1,101 @@
+package metrics
+
+import (
+	"context"
+
+	"strings"
+
+	metrics "github.com/docker/go-metrics"
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/manager/state/store"
+)
+
+var (
+	nodesMetric metrics.LabeledGauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("swarmkit", "", nil)
+	nodesMetric = ns.NewLabeledGauge("nodes", "The number of nodes", metrics.Total, "state")
```

Comment:
I was following convention already set in the engine:
https://github.com/moby/moby/blob/master/daemon/events/metrics.go

Since they're still experimental (and we can still change them), maybe it'd be worth going through and fixing them.